### PR TITLE
Download CSV file about statistic

### PR DESCRIPTION
**Pull request by marla-singer**
Related to PR #2820 
- Most Frequent Users: limit rows, sorting by the highest calls on default, download CSV file #2762
- Errors list table: limit rows, download CSV file #2761

joxi_screenshot_1504095920155

### CODE REVIEW

File: apinf_packages/dashboard/client/analytic/query.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ import moment from 'moment';
 export default function queryForAnalyticPage (requestPath) {
   // Plus one day to include current day in selection
   const today = moment().add(1, 'days').format('YYYY-MM-DD');
-  const oneWeekAgo = moment().subtract(7, 'days').format('YYYY-MM-DD');
-  const twoWeeksAgo = moment().subtract(14, 'days').format('YYYY-MM-DD');
+  const oneWeekAgo = moment().subtract(33, 'days').format('YYYY-MM-DD');
```
Comment (brylie) [inline apinf_packages/dashboard/client/analytic/query.js]: Change variable names to reflect code. I.e. a week is less than 33 days.

File: apinf_packages/dashboard/client/analytic/query.js [hunk 2]
```diff
@@ -10,8 +10,8 @@ import moment from 'moment';
...
+  const twoWeeksAgo = moment().subtract(66, 'days').format('YYYY-MM-DD');
```
Comment (brylie) [inline apinf_packages/dashboard/client/analytic/query.js]: Change variable names to reflect code. I.e. two weeks is less than 66 days.

Also, does moment allow you to `subtract(2, 'months')`?

File: apinf_packages/dashboard/client/analytic/query.js (hunk 1 above)
Comment (brylie) [reply apinf_packages/dashboard/client/analytic/query.js]: Also, does moment allow you to `subtract(1, 'months')`?

Comment (marla-singer) [reply apinf_packages/dashboard/client/analytic/query.js]: No, it is tested values and I forgot to return it back. Thanks

**Comment by marla-singer** [comment]
@brylie Fixed

**Pull request merged**